docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,415 @@
+---
+title: 2021 MUI Developer Survey results
+description: Results for our yearly developer survey, 2021 edition.
```

Comment:
How can we revise the title and subheader to better describe the value of this post to the reader? I'll do some thinking on this and circle back around with some ideas.

Reply:
Some ideas for headlines:
- Uncovering new trends in the 2021 MUI developer survey
- What we learned from the 2021 MUI developer survey
- The 2021 MUI developer survey: here's what we discovered

And some subheaders that could go with them:
- Your feedback helps us to build better products. Here's what we learned about your needs in our latest annual survey.
- Find out what we learned from our latest annual user survey, and how we're applying the results to our 2022 roadmap.
- Here's what we learned in our latest survey, and how we're putting your feedback into action.
- You told us what you needed most, and we've heard you loud and clear! Here's how we're putting your feedback into action.

Reply:
I'm leaning towards: 

> **The 2021 MUI developer survey: here's what we discovered**
> Here's what we learned in our latest survey, and how we're putting your feedback into action.

To share a thought: given that the survey is meant to be a traditional sort of thing, done annually, I believe we should stick to the same title and description across the years. It isn't the usual type of post that we design titles to drive more engagement but rather one that we bet in the consistency aspect (every year we have it) to drive a natural interest in it. There's also the audience aspect to it, as we were discussing in a previous thread. This might be to a very specific, small, audience and not for the general population visiting the docs or even using the library.

Reply:
> we are putting your feedback into action, and in this piece we'll explain how and why.

I agree with the above, however, I would propose that we frame the value to the MUI super-fan at the stage before "putting into action".

The survey is **not** about going into solution mode, I think that this is critically important to keep this marginal. Instead, we try to figure out what are the biggest pain points, what we have improved in the past, what we failed to improve with our actions in the past. It's about 1. synching with reality, 2. bringing transparency to this hence forcing intellectual, 3. and giving direction to the community (and our team) on what matters to be solved next.

So, my proposal here would be to frame it more around you will learn about what the community is experiencing. A s side nice effect is that it show we care about them.

Reply:
That makes sense @oliviertassinari. In that case, I would lean towards this angle:

title: The 2021 MUI developer survey: here's what we discovered
description: Your feedback helps us to build better products. Here's what we learned about your needs in our annual survey.

Reply:
Yup, we're all perfectly on the same page. What I was trying to put across is only that across the years, the Survey post should have the same title+description, only with the year changed. I'm good with the above proposal!

Reply:
Resolving this one.